client/src/boot/BootRoutes.tsx

```diff
@@ -166,13 +162,13 @@ class BootRoutes {
     // Legacy support – make sure we don't overwrite the `onbeforeunload`
     // handler in `LeftAndMain.EditForm.js` but run it after the new handler.
     const currBeforeUnload = window.onbeforeunload;
-    window.onbeforeunload = () => {
+    window.onbeforeunload = (event) => {
```

Comment:
This is a good example of a somewhat annoying and somewhat useful behaviour. Typescript is really good at auto detecting the types of your variables. In this case, TS knows what the default signature of `window.onbeforeunload` is and guess that `currBeforeUnload` will have the same signature. A bit lower, TS was throwing an error because we we calling a method with invalid parameters.

In this context this error is not really applicable because of he nature of the method, but in other scenarios it might identify an actual error.